requirements.txt

```diff
@@ -12,10 +12,11 @@ ipython>=5.4.0,<6.0.0
 statistics
 requests
 rlp>=0.5.1,<0.6.0
-devp2p>=0.8.0
```

Comment:
It seems that `devp2p` is still required if we use `python setup.py install` to install pyethapp?